benchto-service/src/main/webapp/index.html

```diff
@@ -34,7 +34,6 @@
     <script src="js/filters.js"></script>
     <script src="js/main.js"></script>
 
-    <link rel="stylesheet" href="http://google-code-prettify.googlecode.com/svn/trunk/src/prettify.css">
```

Comment:
First, I don't think the app needs this in any way.
Second, http://google-code-prettify.googlecode.com/ directs me to https://github.com/google/code-prettify which instructs me to use `<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>` ... which the app already does (same file, 23 lines earlier)